tests/similarItems.test.ts

```diff
@@ -100,9 +104,33 @@ describe('Gear', () => {
 		body: 'Elite void top'
 	});
 	test('', () => {
+		expect(testGear8.hasEquipped('Scythe of vitur', true)).toBeTruthy();
+		expect(testGear8.hasEquipped('Void knight top', true)).toBeTruthy();
 		expect(testGear8.hasEquipped('Kodai wand', true)).toBeTruthy();
 		expect(testGear8.hasEquipped('Black mask (i)', true)).toBeTruthy();
 		expect(testGear8.hasEquipped('Scythe of vitur', true)).toBeTruthy();
 		expect(testGear8.hasEquipped('Void knight top', true)).toBeTruthy();
 	});
+
+	for (const [baseItem, slot, similarItems] of [
+		['Scythe of vitur', '2h', ['Holy scythe of vitur', 'Sanguine scythe of vitur']],
+		[
+			'Scythe of vitur (uncharged)',
+			'2h',
+			['Holy scythe of vitur (uncharged)', 'Sanguine scythe of vitur (uncharged)']
+		],
+		['Sanguinesti staff', 'weapon', ['Holy sanguinesti staff']],
+		['Sanguinesti staff (uncharged)', 'weapon', ['Holy sanguinesti staff (uncharged)']]
```

Comment:
Should add a charged variant of both Scythe + Sang to the 'Un-dyed (uncharged)' check test, because charged should as an uncharged, but not the other way  obv.